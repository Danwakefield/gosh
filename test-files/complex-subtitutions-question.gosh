EMPTY=
FULL=SUCCESS
echo ${FULL?FAIL}
echo "Next Line Should Be Empty"
echo ${EMPTY?FAIL}
echo ${EMPTY:?SUCCESS}
echo "FAIL: Shell Should Have Exited"
