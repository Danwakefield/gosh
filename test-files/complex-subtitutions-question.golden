SUCCESS
Next Line Should Be Empty

SUCCESS
